### PR TITLE
Supports FRR-VRRP configuration

### PR DESCRIPTION
**Pull request by philo-micas**
##### Work item tracking
- Microsoft ADO **(number only)**:

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211
- [ ] 202305

#### Tested branch (Please provide the tested image version)

- [ ] 
- [ ]

### CODE REVIEW

File: src/sonic-yang-models/yang-models/sonic-vrrp.yang [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        container VRRP {
+
+            description "VRRP part of configdb.json"
+
+            list VRRP_LIST {
+
+                key "interface_name vrid";
+
+                leaf interface_name {
+                    type union {
```
Comment (prvattem) [inline src/sonic-yang-models/yang-models/sonic-vrrp.yang:33]: Please leafref to sonic-interface, sonic-port-channel, sonic-vlan

Comment (philo-micas) [reply src/sonic-yang-models/yang-models/sonic-vrrp.yang:33]: ok.

File: dockers/docker-fpm-frr/frr/supervisord/critical_processes.j2 [hunk 2]
```diff
@@ -1,6 +1,9 @@
 program:zebra
 program:staticd
 program:bgpd
+program:vrrpd
```
Comment (lguohan) [inline dockers/docker-fpm-frr/frr/supervisord/critical_processes.j2:4]: what is the communication channel between vrrpd and zebra, I think we need to put vrrpd into a seperate container, not merging with bgp container. the vrrpd needs to enable/disable separately as feature level. this setup does not allow us to achieve that.

**Review by lguohan — CHANGES_REQUESTED** [review]
put vrrpd into a separate container.

**Pull request closed without merging**